packages/web/package.json

```diff
@@ -17,5 +20,8 @@
   "keywords": [
     "dashx"
   ],
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
+    "uuid-random": "^1.3.2"
```

Comment:
Suggestion: Checkout https://github.com/lukeed/uid

Reply:
We need uuidv4, this is not uuidv4.

But there is a package by same user that we can use: https://github.com/lukeed/uuid
Also the current package is not that slow https://www.npmjs.com/package/uuid-random#performance